### DISCUSSION TITLE
Option to exclude provider from library

### DISCUSSION THREAD

**Discussion by lazyboy0172** [music-assistant/support#2547] [Feature requests and ideas]
I have a very large collection of mashups by various DJs. The organization is a bit of a mess due to the tracks having many artists or being attributed to DJ vs artist, etc. This is set up as a plex library, but when added to Music Assistant it gets added to my home page library as well, meaning my list of artists, albums, etc. balloons into the thousands and is immediately overtaken mostly by these mashups. I'd still like the ability to browse/search/play from this provider, but I'd prefer if it didn't show up in the library of artists, albums, playlists etc. As the list of providers grows over time I expect this will be more of an issue as people add providers and get duplicates or unwanted items on the homepage when really they just want to expand their search pool.

**Comment by OzGav** [comment]
Already a task on the backlog https://github.com/orgs/music-assistant/projects/2/views/1?pane=issue&itemId=62335206

**Comment by OzGav** [comment]
BTW duplicates shouldn't happen as MA identifies those and collapses them together

**Comment by lazyboy0172** [comment]
Ah thanks so much. I know duplicates aren’t supposed to happen but also know that’s not 100%. I didn’t see the previous request though, sorry. I kept searching for terms like hide and library and that particular thread didn’t show up. Thanks again

**Comment by OzGav** [comment]
This has been implemented in the 2.7.0 beta

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
